pkg/handler/handler.go

```diff
@@ -293,14 +293,33 @@ func (h handler) update(w http.ResponseWriter, r *http.Request, params map[strin
 	}
 
 	var req *broker.UpdateRequest
+
 	if err := readRequest(r, &req); err != nil {
 		writeResponse(w, http.StatusBadRequest, broker.ErrorResponse{Description: err.Error()})
 		return
 	}
 
-	resp, err := h.broker.Update(instanceUUID, req)
+	var async bool
 
-	writeDefaultResponse(w, http.StatusOK, resp, err)
+	// ignore the error, if async can't be parsed it will be false
+	async, _ = strconv.ParseBool(r.FormValue("accepts_incomplete"))
```

Comment:
We must be checking for auto escalate and checking for users access to the namespace. see: https://github.com/openshift/ansible-service-broker/blob/master/pkg/handler/handler.go#L246

Reply:
I'll take a look at this, thanks for pointing it out @shawn-hurley